src/Raven.Client/Documents/Conventions/JsonNetBlittableEntitySerializer.cs

```diff
@@ -15,8 +15,9 @@ internal class JsonNetBlittableEntitySerializer
         private static BlittableJsonReader _reader;
         [ThreadStatic]
         private static JsonSerializer _serializer;
+        [ThreadStatic]
+        private static Action<JsonSerializer> _customize;
```

Comment:
This is wrong, we can't use `ThreadStatic` here, we are better off using `ThreadLocal`, because we may have mutiple different serializers